Yoast/javascript
# Lin 495 add function words for arabic

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds Arabic function words.
* [Yoast SEO Free] Improves keyphrase recognition in Arabic by filtering the function words such as `أنا`, `صفر`, `الـ`.

## Test instructions

This PR can be tested by following these steps:

Test the following in WP SEO Free:
* Set the site language to Arabic.
* Create a new post and add a text with at least 150 words.
* Set your keyphrase to `هذان الزوجان`.
* Add the word `الزوجان` to the text.
* Make sure that the keyphrase density shows that the keyphrase is found once.

Test instructions for Lingo team:
* Run yarn test and make sure all current specs pass
* Run the Content analysis app and set the language to Arabic 
* Create a new post and add a test with at least 150 words.
* Set your keyphrase to  `هذين المحققين`.
* Add the word `المحققين` to the text
* Make sure that the keyphrase density shows that the keyphrase is found once.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-495
